src/scripts/main.js

```diff
@@ -1,29 +1,58 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const validTypes = ['success', 'error', 'warning'];
+
+  if (!validTypes.includes(type)) {
+    throw new Error(
+      `Invalid notification type: "${type}". Expected one of ${validTypes.join(', ')}.`,
+    );
+  }
+
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+  div.style.top = `${posTop}px`;
+  div.style.right = `${posRight}px`;
+
+  const titles = document.createElement('h2');
+
+  titles.className = 'title';
+  titles.textContent = title;
+  div.appendChild(titles);
+
+  const descriptions = document.createElement('p');
+
+  descriptions.textContent = description;
+  div.appendChild(descriptions);
+
+  document.body.appendChild(div);
+
+  setTimeout(() => {
+    div.style.display = 'none';
+  }, 2000);
 };
 
 pushNotification(
   10,
   10,
   'Title of Success message',
-  'Message example.\n ' + 'Notification should contain title and description.',
+  `Message example. Notification should contain title and description.`,
   'success',
 );
```

Comment:
You are calling the function `pushNotification` multiple times with different parameters. This is fine, but if you find yourself doing this frequently, you might want to consider creating a higher-level function that can generate these notifications with less repetition.